app/org/sagebionetworks/bridge/dynamodb/DynamoSurveyDao.java

```diff
@@ -303,6 +310,7 @@ public void deleteSurvey(StudyIdentifier studyIdentifier, GuidCreatedOnVersionHo
     public Survey closeSurvey(GuidCreatedOnVersionHolder keys) {
```

Comment:
Questions about making this work like consents: every save is a new copy. Everything is then immutable, and never deleted. This is a lot of data from tests that is never deleted.